.env.example

```diff
@@ -1,5 +1,6 @@
 # View & likes tracking 
 FAUNA_SECRET=
+DATABASE_URL="postgresql://clarence:@localhost:5432/theodorusclarence"
```

Comment:
Is this okay? This expose your local db is passwordless

Reply:
It's all good, I purposefully removed the db password in the env example